coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -57,7 +57,7 @@ def get_form_kwargs(self):
         kwargs = super(UpdateAllocationView, self).get_form_kwargs()
         kwargs["user_id"] = self.request.user.id
 
-        allocation_id = self.kwargs.get("allocation_id")
+        kwargs["allocation_id"] = allocation_id = self.kwargs.get("allocation_id")
         allocation = Allocation.objects.get(pk=allocation_id)
```

Comment:
Just passing status_name forward saves work for the form init
```suggestion
        allocation_id = self.kwargs.get("allocation_id")
        allocation = Allocation.objects.get(pk=allocation_id)
        kwargs["status_name"] = allocation.status.name
```